### PR TITLE
feat: simplify StoredNotifAutopushUser's on_start

### PR DESCRIPTION
**Pull request by pjenvey**
to match AutopushUser's quicker initializion time when locust is spawning

and setup a quick matching shape class

Issue: SYNC-3916

### CODE REVIEW

File: tests/load/locustfiles/load.py [hunk 1]
```diff
@@ -85,3 +86,11 @@ def tick(self) -> TickTuple | None:
         spawn_rate: float = max(abs(users - self.get_current_user_count()), 1)
 
         return users, spawn_rate, self.user_classes
+
+
+class StoredNotifAutopushLoadTestShape(AutopushLoadTestShape):
```
Comment (pjenvey) [inline tests/load/locustfiles/load.py:91]: Adding `StoredNotifAutopushUser` to `AutopushLoadTestShape` seemed to force both classes usage during the load test. So I setup this subclass for now so we can focus on solely `StoredNotifAutopushUser`, we can revisit this later.

File: tests/load/locustfiles/load.py [hunk 2]
```diff
@@ -85,3 +86,11 @@ def tick(self) -> TickTuple | None:
...
+    """A load test shape class for StoredNotifAutopushLoadTestShape
+    (Duration: 10 minutes, Users: 15000).
```
Comment (jrbenny35) [inline tests/load/locustfiles/load.py:93]: Is this supposed to be ```150000```?

File: tests/load/locustfiles/stored.py [hunk 3]
```diff
@@ -184,23 +193,23 @@ def connect_and_unsubscribe(self):
...
         """connect_and_hello then disconnect"""
+        if not self.initialized:
+            logger.debug("Task 'connect_and_read' skipped.")
+            return
         self.connect_and_hello()
         self.close()
 
-    def connect_and_register(self) -> None:
-        """Initialize the WebSocket and Hello/Register initial channels"""
+    def connect(self) -> None:
```
Comment (jrconlin) [inline tests/load/locustfiles/stored.py:205]: Can we add a comment to this to explain what this is really doing? It took me a bit to work it out.

**Review by jrconlin — APPROVED** [review]

**Review by jrconlin — APPROVED** [review]

**Review by jrbenny35 — APPROVED** [review]
After testing this a few times, I think this looks good and is usable. Thanks Phil!

**Pull request merged**